test/suite/test_s3_store01.py

```diff
@@ -44,7 +45,7 @@ class test_s3_store01(wttest.WiredTigerTestCase):
     # Bucket name can be overridden by an environment variable.
     bucket_name = os.getenv('WT_S3_EXT_BUCKET')
     if bucket_name is None:
-        bucket_name = "s3testext"
+        bucket_name = "wt-bucket"
```

Comment:
Please keep the default bucket "s3testext" - that is the one automated testing is expecting.